Lib/test/test_ctypes/test_callbacks.py

```diff
@@ -149,8 +149,9 @@ def callback(a, b):
             return c
         dll = cdll[_ctypes_test.__file__]
         # With no fix for i38748, the next line will raise OSError and cause the test to fail.
-        self.assertEqual(dll._test_i38748_runCallback(callback, 5, 10), 15)
-
+        with support.captured_stdout() as out:
```

Comment:
Move it one line above.

Reply:
I am not sure that I understand you correctly, please look at the last change

Reply:
I meant that the comment refers to the "next line", so it should be inside the with block.